mods/cnc/maps/funpark01/scj01ea.lua

```diff
@@ -0,0 +1,82 @@
+RifleReinforcments = { "e1", "e1", "e1", "bike" }
+BazookaReinforcments = { "e3", "e3", "e3", "bike" }
+BikeReinforcments = { "bike" }
+
+
+ReinforceWithLandingCraft = function(units, transportStart, TransportUnload, rallypoint)
+	local transport =  Actor.Create("oldlst", true, { Owner = player, Facing = 0, Location = transportStart })
+	local subcell = 0
+	Utils.Do(units, function(a)
+		transport.LoadPassenger(Actor.Create(a, false, { Owner = transport.Owner, Facing = transport.Facing, Location = transportUnload, Subcell = subcell }))
+		subcell = subcell + 1
+	end)
+	
+	transport.ScriptedMove(transportUnload)
+	
+	transport.CallFunc(function()
+		Utils.Do(units, fucntion()
+			local a = transport.UnloadPassenger()
+			a.IsinWorld = true
+			a.MoveIntoWorld(transport.Location - CVec.New(0, 1))
+			
+			if rallypoint ~= nil then
+				a.Move(rallypoint)
+			end
+		end)
+	end)
+	
+	transport.Wait(5)
+	transport.ScriptedMove(TransportStart)
+	transport.Destroy()
+	
+	Media.PlaySpeechNotification(player, "Reinforce")
+end
+
+WorldLoaded = function()
+	nod = Player.GetPlayer("Nod")
+	dinosaur = Player.GetPlayer("Dinosaur")
+	
+	NodObjective1 = nod.AddPrimaryObjective("Investigate the nearby village for reports of /nstrange activity")
+	
+	Trigger.OnObjectiveCompleted(nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+
+	Trigger.OnObjectiveFailed(nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+	
+	Trigger.OnPlayerWon(nod, function()
+		Media.PlaySpeechNotification(nod, "Win")
+	end)
+
+	Trigger.OnPlayerLost(nod, function()
+		Media.PlaySpeechNotification(nod, "Lose")
+	end)
+	
+	DinoTric.Patrol({WP0.Location, WP1.Location}, true, 3)
+	DinoTrex.Patrol({WP2.Location, WP3.Location}, false)
+	Trigger.OnIdle(DinoTrex, DinoTrex.Hunt)
+
+	function(
+		ReinforceWithLandingCraft(RifleReinforcments, EntryA.Location, ReinforceA.Location, ReinforceA.Location)
+		InitialUnitsArrived = true
```

Comment:
Imho it would be better to have:
```
ReinforceWithLandingCraft(RifleReinforcments, EntryA.Location, ReinforceA.Location, ReinforceA.Location)

Trigger.AfterDelay(DateTime.Seconds(15), function()
	ReinforceWithLandingCraft(BazookaReinforcments, EntryB.Location, ReinforceB.Location, einforceB.Location)
	InitialUnitsArrived = true
end)
```